Source/common/SNTConfigurator.h

```diff
@@ -371,6 +371,22 @@
 ///
 @property(readonly, nonatomic) BOOL fcmEnabled;
 
+///
+/// True if metricsFormat and metricsURL are set. False otherwise.
+///
+@property(readonly, nonatomic) BOOL exportMetrics;
+
+///
+/// String format for to export metrics as, defaults to "".
+///
+@property(readonly, nonatomic) NSString *metricFormat;
```

Comment:
It might be better for this to be an enum with the getter handling the conversion from a string. That way downstream code can switch on it.

Reply:
I switched it to an enum. Originally I didn't do this because I was thinking that its less descriptive in the config. 

However since we're already doing this for ClientMode I agree it makes more sense as an enum.